libs/etterlatte-funksjonsbrytere/src/main/kotlin/no/nav/etterlatte/funksjonsbrytere/FeatureToggleService.kt

```diff
@@ -67,10 +50,12 @@ class DummyFeatureToggleService : FeatureToggleService {
         )
     }
 
-    private val overstyrteBrytere = mapOf(
-        Pair(FellesFeatureToggle.NoOperationToggle, true)
+    private val overstyrteBrytere = mutableMapOf<FeatureToggle, Boolean>(
+        FellesFeatureToggle.NoOperationToggle to true
     )
 
+    fun settBryter(bryter: FeatureToggle, verdi: Boolean) = overstyrteBrytere.put(bryter, verdi)
+
     override fun isEnabled(toggleId: FeatureToggle, defaultValue: Boolean) =
         overstyrteBrytere.getOrDefault(toggleId, true)
```

Comment:
ehh, er den eksisterende koden for `isEnabled` ikke helt bra?

Reply:
Hehe, verdt å få med at dette er i `DummyFeatureToggleService`-klassa, altså den vi kun bruker for test. Når eg tenkjer meg om kunne vi kanskje skrive om litt og så flytta den til testdata-modulen 🤔